src/foam/nanos/theme/Theme.js

```diff
@@ -206,8 +206,9 @@ foam.CLASS({
       class: 'String',
       name: 'loginView',
       documentation: `Border used to wrap unauthenticated views like loginViews. Set to nullBorder for no additional borders`,
-      value: 'foam.u2.borders.BaseUnAuthBorder'
+      value: 'foam.nanos.auth.login.LoginView'
```

Comment:
This should be BaseUnAuthBorder as the loginView will always be the one LoginView we use across apps. This property should only be the border that is used around the loginView